public/app/features/alerting/unified/mocks/server/handlers/mimirRuler.ts

```diff
@@ -1,9 +1,25 @@
 import { delay, http, HttpResponse } from 'msw';
 
-import { RulerRuleGroupDTO } from '../../../../../../types/unified-alerting-dto';
+import {
+  PromRulesResponse,
+  RulerRuleGroupDTO,
+  RulerRulesConfigDTO,
+} from '../../../../../../types/unified-alerting-dto';
 import { namespaces } from '../../mimirRulerApi';
 import { HandlerOptions } from '../configure';
 
+export const getRulerRulesHandler = () => {
+  return http.get(`/api/ruler/:dataSourceUID/api/v1/rules`, async () => {
+    return HttpResponse.json<RulerRulesConfigDTO>(namespaces);
+  });
+};
+
+export const prometheusRulesHandler = () => {
+  return http.get('/api/prometheus/:dataSourceUID/api/v1/rules', () => {
+    return HttpResponse.json<PromRulesResponse>({ status: 'success', data: { groups: [] } });
+  });
+};
```

Comment:
Something for us to be wary of is our handlers overwriting each other - if we accidentally reordered the handlers in `all-handlers` and put this one first, it would match requests before the grafana specific one. Just something to keep in mind later on (could sneak through before it started mattering, as the responses are the same ATM)